firstvoices/backend/serializers/base_serializers.py

```diff
@@ -122,7 +139,7 @@ class LinkedSiteSerializer(serializers.HyperlinkedModelSerializer):
         view_name="api:site-detail", lookup_field="slug"
     )
     language = serializers.StringRelatedField()
-    visibility = serializers.CharField(read_only=True, source="get_visibility_display")
+    visibility = serializers.SerializerMethodField(read_only=True)
```

Comment:
Can't this be removed entirely, since the mixin defines it?

Reply:
I did try to remove these lines but without the visibility being defined as a SerializerMethodField DRF detects/displays visibility as an integer

Reply:
Hmm, there should be a way to solve that. Can you have a look at the RelatedMediaSerializerMixin? It manages to define fields that don't have to be redefined on the consuming serializers

Reply:
Will do!